packages/uui-base/lib/mixins/SelectableMixin.ts

```diff
@@ -80,10 +80,16 @@ export const SelectableMixin = <T extends Constructor<LitElement>>(
     }
 
     private handleSelectKeydown = (e: KeyboardEvent) => {
-      //if (e.composedPath().indexOf(this.selectableTarget) !== -1) {
-      if (this.selectableTarget === this) {
-        if (e.key !== ' ' && e.key !== 'Enter') return;
-        this._toggleSelect();
+      const composePath = e.composedPath();
+      if (
+        (this._selectable || (this.deselectable && this.selected)) &&
+        composePath.indexOf(this.selectableTarget) === 0
+      ) {
+        if (this.selectableTarget === this) {
+          if (e.key !== ' ' && e.key !== 'Enter') return;
```

Comment:
Can we use e.code !== 'Space' here instead? I think that is safer than the empty space.

Reply:
good input